test/operators.jl

```diff
@@ -242,6 +242,21 @@
         end
     end
 
+    gx = SimpleGraph(100, 200)
+    @testset "Graph product edge counts: $g" for g in testgraphs(gx)
+        gy = SimpleGraph(123, 234)
+        v1 = nv(gx)
+        v2 = nv(gy)
+        e1 = ne(gx)
+        e2 = ne(gy)
+        # Edge counts from https://en.wikipedia.org/wiki/Graph_product
+        @test ne(cartesian_product(gx, gy)) == v1*e2 + v2*e1
+        @test ne(tensor_product(gx, gy)) == 2*e1*e2
+        @test ne(lexicographic_product(gx, gy)) == v1*e2 + e1*v2^2
+        @test ne(strong_product(gx, gy)) == v1*e2 + v2*e1 + 2*e1*e2
+        @test ne(disjunctive_product(gx, gy)) == v1^2*e2 + e1*v2^2 - 2*e1*e2
+    end
```

Comment:
`g` is not used inside the loop. It should be at the place of gx. Moreover this should be done also for gy.
I suggest these changes:

```suggestion
    gx = SimpleGraph(100, 200)
    gy = SimpleGraph(123, 234)
    @testset "Graph product edge counts" for (g1,g2) in zip(testgraphs(gx),testgraphs(gy))
        v1 = nv(g1)
        v2 = nv(g2)
        e1 = ne(g1)
        e2 = ne(g2)
        # Edge counts from https://en.wikipedia.org/wiki/Graph_product
        @test ne(cartesian_product(g1, g2)) == v1*e2 + v2*e1
        @test ne(tensor_product(g1, g2)) == 2*e1*e2
        @test ne(lexicographic_product(g1, g2)) == v1*e2 + e1*v2^2
        @test ne(strong_product(g1, g2)) == v1*e2 + v2*e1 + 2*e1*e2
        @test ne(disjunctive_product(g1, g2)) == v1^2*e2 + e1*v2^2 - 2*e1*e2
    end
```